### DISCUSSION TITLE
Cannot action send_goal /coverage in create3_examples

### DISCUSSION THREAD

**Discussion by nagvajara** [iRobotEducation/create3_docs#187] [Questions] [answered]
I did name remapping for one robot whose namespace is /yeti

ros2 run create3_coverage create3_coverage --ros-args -r __ns:=/yeti

/yeti/coverage [create3_examples_msgs/action/Coverage] populates when
I do ros2 action list --show-type

however, I got
The passed action type is invalid when
ros2 action send_goal /yeti/coverage create3_examples_msgs/action/Coverage "{explore_duration:{sec: 500, nanosec: 0}, max_runtime:{sec: 1000,nanosec: 0}}"

**Comment by justinIRBT** [comment, marked as answer]
That error message indicates you don't have the messages in your path.  Did you source the `install/setup.bash` in the sandbox where you built the create3_examples_msgs?

**Reply by nagvajara** [reply]
Problem solved by sourcing local_setup.bash sourcing setup.bash not sufficient.
thank you
